app/src/app/[lng]/[inventory]/data/[step]/[subsector]/page.tsx

```diff
@@ -129,14 +134,17 @@ function SubSectorPage({
   };
 
   const getFilteredSubsectorScopes = () => {
+    if (!inventoryData) return [];
     return Object.entries(MANUAL_INPUT_HIERARCHY)
       .filter(([key]) => key.startsWith(subSectorData?.referenceNumber!))
-      .map(([k, v]) => ({
-        ...v,
-        referenceNumber: k,
-      }));
+      .map(([k, v]) => ({ ...v, referenceNumber: k }))
+      .filter((scope) =>
+        getScopesForInventoryAndSector(
+          inventoryData.inventoryType!,
```

Comment:
Can we do a null check beforehand and throw a more useful error here in case it is null-ish?

Reply:
there should never be a case where it's null because it has a default setting in the DB. and also I'm checking that inventoryData is there

Reply:
Ok great, then we can adjust the type definition in the model file so it's clear that it's not null? We can do it in a follow-up tech debt ticket next sprint if necessary.

Reply:
https://openearth.atlassian.net/browse/ON-2851